linchpin/tests/mockdata/dummy/fetch/ws2/LICENSE

```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```

Comment:
We can't be confusing licenses. Probably ought to use a different workspace. Maybe a simpler dummy workspace.